daprdocs/content/en/python-sdk-docs/_index.md

```diff
@@ -9,46 +9,142 @@ no_list: true
 
 Dapr offers a variety of packages to help with the development of Python applications. Using them you can create Python clients, servers, and virtual actors with Dapr.
 
-## Available packages
+## Prerequisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [Python 3.7+](https://www.python.org/downloads/) installed
 
-- [**Dapr client**]({{< ref python-client.md >}}) for writing Python applications to interact with the Dapr sidecar and other Dapr applications
-- [**Dapr actor**]({{< ref python-actor.md >}}) for creating and interacting with stateful virtual actors in Python
-- [**Extensions**]({{< ref python-sdk-extensions >}}) for adding Dapr capabilities to other Python frameworks
-    - [**gRPC extension**]({{< ref python-grpc.md >}}) for creating a gRPC server with Dapr
-    - [**FastAPI extension**]({{< ref python-fastapi.md >}}) for adding Dapr actor capabilities to FastAPI applications
-    - [**Flask extension**]({{< ref python-flask.md >}}) for adding Dapr actor capabilities to Flask applications
+## Installation
 
-## Install the Dapr module
+To get started with the Python SDK, install the main Dapr Python SDK package. 
 
 {{< tabs Stable Development>}}
 
 {{% codetab %}}
+<!--stable-->
 ```bash
 pip install dapr
 ```
 {{% /codetab %}}
 
 {{% codetab %}}
-{{% alert title="Note" color="warning" %}}
-The development package will contain features and behavior that will be compatible with the pre-release version of the Dapr runtime. Make sure to uninstall any stable versions of the Python SDK before installing the dapr-dev package.
+<!--dev-->
+> **Note:** The development package will contain features and behavior that will be compatible with the pre-release version of the Dapr runtime. Make sure to uninstall any stable versions of the Python SDK before installing the dapr-dev package.
 {{% /alert %}}
 
 ```bash
 pip install dapr-dev
 ```
+
 {{% /codetab %}}
 
 {{< /tabs >}}
 
+
+## Available packages
+
+Once you've installed the main Python SDK, you can:
+- Import additional packages as part of the main SDK package
+- Install extensions on top of the main Python SDK
```

Comment:
Non-blocking: Technically when you install the extensions it will install the main SDK anyway because that's a dependency of those extensions.